### PR TITLE
This should include the latest roles

### PR DESCRIPTION
**Pull request by mgifford**
Go to the [new demo site.](https://federalist-6d3494a6-5b03-42fa-82f2-57f3bbb203af.app.cloud.gov/preview/gsa/openacr-website/create/)

Verify:
- Main menu items show Home, Samples, About, & Roles
- New Roles submenu includes Procurement, Vendors, Accessibility SME, & Other
- Secondary navigation (above search) goes to /vpat/ & /acr/
- Check that the Accessibility Statement link goes to the new footer page
- Check highlight text & links on the home page
- Review new pages for obvious errors:
    - /procurement/
    - /accessibility-expert/
    - /vendors/
    - /others/
    - /accessibility/
    - /vpat/
    - /acr/
    - /roles/

Grumbles: I needed to delete the old branch. Kinda fumbled this commit after working away on the content over the afternoon.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -93,12 +91,25 @@ primary_navigation:
         url: /roadmap/
       - name: Resources
         url: /resources/
+  - name: Roles
+    url: /roles/
+    children:
+      - name: Procurement
+        url: /procurement/
+      - name: Vendors
+        url: /vendoor/
```
Comment (dmundra) [inline _config.yml:100]: Spelling of the permalink `vendoor`

File: _config.yml [hunk 2]
```diff
@@ -93,12 +91,25 @@ primary_navigation:
...
+      - name: Accessibility SME
+        url: /accessibility-expert/
+      - name: Other
+        url: /other/
```
Comment (dmundra) [inline _config.yml:104]: URL here is singular `other` while the page itself is plural `others`

File: _pages/other.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Others
+layout: page
+sidenav: false
+permalink: /others/
```
Comment (dmundra) [inline _pages/other.md:5]: I think this should changed to singular or update the filename and the child array to match this one.

**Review by dmundra — CHANGES_REQUESTED** [review]
@mgifford I think I found why the htmlproofer is failing. Rest of the changes would probably good to have @danid123 review.

File: _pages/vendor.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: Vendor Team
+layout: page
+sidenav: false
+permalink: /vendors/
```
Comment (dmundra) [inline _pages/vendor.md:5]: @mgifford another singular/plural mismatch. Here it is `vendors` but the child menu link is `vendor`

**Comment by danid123** [comment]
@mgifford The content looks pretty good. The only thing that seems a little awkward is the Others section for Roles. Were you thinking that there would be more roles listed there? Otherwise I would just change it to match the headline.

**Comment by mgifford** [comment]
I agree that the others role is awkward.  Who wants to be an other anyways. 

I think that there are other groups though. Could be accessibility organizations. Affinity groups inside of organizations. 

I need to scan through the list of people we highlighted as part of our personas & possible audiences to see if we missed anyone.  

I ultimately broke it into these sections as it seemed to make more sense than giving each role their own page. At least for now.

**Comment by danid123** [comment]
Maybe instead of Others it could be Advocates? Or Advocates for People with Disabilities?

**Comment by mgifford** [comment]
That makes sense. I'll make the changes from others.md, others, Others

to something like advocates.md, advocates & Advocates for Inclusion

"Advocates for People with Disabilities" is just a mouthful.

**Comment by mgifford** [comment]
> @mgifford I think I found why the htmlproofer is failing. Rest of the changes would probably good to have @danid123 review.

Are there changes that are needed here @dmundra

**Comment by dmundra** [comment]
> > @mgifford I think I found why the htmlproofer is failing. Rest of the changes would probably good to have @danid123 review.
> 
> Are there changes that are needed here @dmundra

Nope you are good on that front!

**Review by danid123 — APPROVED** [review]
@mgifford I think it looks great. Sometime in the future we might consider adding Agency Section 508 Program Managers (PMs)?

**Review by dmundra — APPROVED** [review]
Approved!

**Pull request merged**